### PR TITLE
Williamhe/1735613 remove 3.11.1 references from codebase

### PR DESCRIPTION
**Pull request by william-msft**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference a
  - Removing all references of 3.11.1 in Oryx code base for Oryx-CI release
  - Removed 3.11.1 from dev storage account as well
- [x] Tests are included and/or updated for code changes.
~- [ ] Proper license headers are included in each file.~

### CODE REVIEW

File: build/__pythonVersions.sh [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-PYTHON_RUNTIME_BASE_TAG='20230208.7'
+PYTHON_RUNTIME_BASE_TAG='20230209.1'
```
Comment (william-msft) [inline build/__pythonVersions.sh:3]: This base tag is from an Oryx-BaseImages-Python pipeline invocation that ran with 3.11.1 removed.

File: tests/Oryx.Integration.Tests/Python/PythonEndToEndTests.cs [hunk 2]
```diff
@@ -66,7 +66,7 @@ public async Task CanBuildAndRun_Tweeter3AppAsync()
                 });
         }
 
-        [Fact(Skip = "Story #1735613 we want to wait on releasing latest 3.11 to production but also want to have an Oryx-CI release.")]
+        [Fact]
```
Comment (william-msft) [inline tests/Oryx.Integration.Tests/Python/PythonEndToEndTests.cs:69]: Since we've removed 3.11.1 from our codebase, we can use this test

**Review by daniv-msft — DISMISSED** [review]
Looks good to me! Thanks

**Review by pauld-msft — APPROVED** [review]
:shipit:

**Pull request merged**